Services/source/updater/updater.c

```diff
@@ -12,14 +12,33 @@
 #include "bl_flash.h"
 #include "privileged_functions.h"
 
+//for testing only. do hex dump
```

Comment:
Can you put our header here along with block comments for all functions

Reply:
our header is the copyright correct?

Reply:
yeah, plus file name, author, date -- just copy from another C file